ios/RNRadar.m

```diff
@@ -306,6 +306,10 @@ - (void)didLogMessage:(NSString *)message {
     [Radar stopTracking];
 }
 
+RCT_EXPORT_METHOD(setForegroundServiceOptions) {
+    // not implemented
```

Comment:
Should this throw an error or something until I implement it? Or just leave out these three lines entirely?

Reply:
It shouldn't throw an error, it can just no-op silently. That way the developer doesn't need to add any platform checks in the cross-platform JS code.